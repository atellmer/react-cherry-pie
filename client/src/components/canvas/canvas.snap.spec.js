import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

import TmCanvasDesktop from './desktop';
import TmCanvasPhone from './phone';


 const props = {
    params: {},
    messagePanelHeight: 0,
    changeMessagePanelHeight: () => {}
  };

test('Jest: TmCanvasDesktop (Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();
  const tree = shallowRenderer.render(<TmCanvasDesktop {...props}/>);

  expect(tree).toMatchSnapshot();
});

test('Jest: TmCanvasPhone (Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();
  const tree = shallowRenderer.render(<TmCanvasPhone {...props}/>);

  expect(tree).toMatchSnapshot();
});
