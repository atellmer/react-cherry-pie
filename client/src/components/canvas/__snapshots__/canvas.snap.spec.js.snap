// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest: TmCanvasDesktop (Snapshot) 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
    style={
      Object {
        "paddingBottom": "",
      }
    }
  >
    <Scrollbars
      autoHeight={false}
      autoHeightMax={200}
      autoHeightMin={0}
      autoHide={true}
      autoHideDuration={200}
      autoHideTimeout={1000}
      className={undefined}
      hideTracksWhenNotNeeded={false}
      renderThumbHorizontal={[Function]}
      renderThumbVertical={[Function]}
      renderTrackHorizontal={[Function]}
      renderTrackVertical={[Function]}
      renderView={[Function]}
      tagName="div"
      thumbMinSize={30}
      universal={false}
    >
      <div>
        Route: undefined
      </div>
    </Scrollbars>
  </div>
  <div
    className={undefined}
  >
    <TmMessagePanel
      changeMessagePanelHeight={[Function]}
    />
  </div>
</div>
`;

exports[`Jest: TmCanvasPhone (Snapshot) 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
    style={
      Object {
        "paddingBottom": 0,
      }
    }
  >
    <Scrollbars
      autoHeight={false}
      autoHeightMax={200}
      autoHeightMin={0}
      autoHide={true}
      autoHideDuration={200}
      autoHideTimeout={1000}
      className={undefined}
      hideTracksWhenNotNeeded={false}
      renderThumbHorizontal={[Function]}
      renderThumbVertical={[Function]}
      renderTrackHorizontal={[Function]}
      renderTrackVertical={[Function]}
      renderView={[Function]}
      tagName="div"
      thumbMinSize={30}
      universal={false}
    >
      <div>
        Route: undefined
      </div>
    </Scrollbars>
  </div>
  <div
    className={undefined}
  >
    <TmMessagePanel
      changeMessagePanelHeight={[Function]}
    />
  </div>
</div>
`;
